### DISCUSSION TITLE
Makes 3 Smokes in a row

### DISCUSSION THREAD

**Discussion by Wullff91** [isid0re/SoloLeveling#212] [Q&A]
Hey my sorc just make  3 smoke rws and dont use ist because it isnt better then the two others from her and the merc.
Can i change any pickit or other stuff to stop this ?

**Comment by isid0re** [comment]
You can lower the tier rating in the sorceress.sololeveling.js config file

**Reply by Wullff91** [reply]
ok tahnks
